src/otx/v2/adapters/torch/mmengine/engine.py

```diff
@@ -44,131 +77,108 @@ class MMXEngine(Engine):
     def __init__(
         self,
         work_dir: str | Path | None = None,
-        config: dict | (Config | str) | None = None,
     ) -> None:
         """Initialize a new instance of the MMEngine class.
 
         Args:
             work_dir (Optional[Union[str, Path]], optional): The working directory for the engine. Defaults to None.
-            config (Optional[Union[dict, Config, str]], optional): The configuration for the engine. Defaults to None.
         """
         super().__init__(work_dir=work_dir)
         self.runner: Runner
         self.latest_model = {"model": None, "checkpoint": None}
         self.registry = MMEngineRegistry()
-        self._initial_config(config)
+        self.default_config = Config(DEFAULT_CONFIG)
```

Comment:
How about setting this as a class variable instead of `self.default_config`? This may not be updated and fed into `get_value_from_config` as default.

Reply:
Thanks for the comment. I've changed the variable to declare it as a class static variable and use it, and also changed `get_value_from_config` to `MMXEngine._get_value_from_config`.

Reply:
thank you.